docs/system-overview.rst

```diff
@@ -0,0 +1,79 @@
+Exam System Overview
+===========================
+This document outlines the components involved in the edX exam system. It should
+serve as a guide to orient developers on what components exist and how they interact.
+
+.. contents::
+
+System Components
+-----------------
+
+Interface with edX Systems
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. image:: images/components-internal.png
+
+Interface with Proctoring Tools
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. image:: images/components-external.png
+
+Learning MFE
+^^^^^^^^^^^^
+The user facing web application for edX courseware. This app handles fetching course content from
+the LMS and rendering it within the course structure. For exam sections, the special-exams-lib injects
+its own behavior into this rendering process.
+
+special-exams-lib
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The special-exams React library is responsible for rendering different views and messaging
+depending on the learner's exam attempt state. The goal of these views, or interstitials,
+is to guide the learner through the many steps of setting up, attempting, and submitting
+an exam. At many steps there are critical details about what the learner needs to do or understand.
+
+If the learner is in the appropriate state this library will allow the learning MFE to
+fetch and render the exam questions. In the case of proctored exams, the proctoring tool
+must be successfully launched before exam content can be viewed.
+
+https://github.com/edx/frontend-lib-special-exams/
+
+edx-exams
+^^^^^^^^^
+This service handles all persistent proctoring and exam experience logic. It holds to models
+for exam configuration, learner attempts, and proctoring providers. This service will also
+act as an LTI consumer via the xblock-lti-consumer library building all LTI messages and
+implementing the required callback URLs to complete the launch flow.
+
+LTI Launch Flow
+^^^^^^^^^^^^^^^
+A series of handshakes that ensures the proctoring tool is launched and authenticated with the correct
+context about the learner and the exam. For more details on the specifics of the lauch flow see `TODO LTI DOC <todo>`_
+
+Assessment Control Service API
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The Assessment Control Service (ACS) interface may be used by the proctoring provider to update an
```

Comment:
It might be worth specifying that the `LTI Launch Flow` is mediated by the user's browser, whereas the calls to the `Assessment Control Service API` are done outside the context of a user's browser through web-service calls.